src/app/Plans/components/Wizard/VMConcernsIcon.tsx

```diff
@@ -2,15 +2,28 @@ import * as React from 'react';
 import { IVMwareVM } from '@app/queries/types';
 import { StatusIcon, StatusType } from '@konveyor/lib-ui';
 import { InfoCircleIcon } from '@patternfly/react-icons';
-import { getMostSevereVMConcern } from './helpers';
+import { Flex, FlexItem, Spinner } from '@patternfly/react-core';
+import './VMConcernsIcon.css';
 
+import { getMostSevereVMConcern } from './helpers';
 interface IVMConcernsIconProps {
   vm: IVMwareVM;
 }
 
 const VMConcernsIcon: React.FunctionComponent<IVMConcernsIconProps> = ({
   vm,
 }: IVMConcernsIconProps) => {
+  if (vm.revisionAnalyzed < vm.revision) {
+    return (
+      <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+        <FlexItem>
+          <Spinner aria-valuetext="Analysing" size="sm" className="spinner" />
+        </FlexItem>
+        <FlexItem>Analysing</FlexItem>
```

Comment:
@vconzola should we use "Analysing" or "Analyzing" with a z? I think with a z is the American spelling and everywhere else in the world spells it with an s.